featuretools/entityset/relationship.py

```diff
@@ -65,3 +73,35 @@ def parent_variable(self):
     def child_variable(self):
         """Instance of variable in child entity"""
         return self.child_entity[self._child_variable_id]
+
+    def parent_name(self):
+        """The name of the parent, relative to the child."""
+        if self._is_unique():
+            return self._parent_entity_id
+        else:
+            return '%s[%s]' % (self._parent_entity_id, self._child_variable_id)
+
+    def child_name(self):
+        """The name of the child, relative to the parent."""
+        if self._is_unique():
+            return self._child_entity_id
+        else:
+            return '%s[%s]' % (self._child_entity_id, self._child_variable_id)
+
+    def get_arguments(self):
```

Comment:
Ah, I wasn't aware of that. Yes we definitely only need one. I think there are three considerations:

1. Which dict/JSON structure to use? `get_arguments` is more explicit I think, but if we replace `relationship_to_description` with this it changes the entityset serialization format. Is compatibility a concern there?
2. Should the method live in `entityset/serialize.py`, or on `Relationship`? Since it's now being used in multiple places I would lean towards `Relationship`.
3. Which name to use? I chose `get_arguments` to match what we did for features. But I don't think it's that clear without context – "arguments" for what?

Reply:
1. let's use what you have currently for `get_arguments` and update the entityset serialization format. we don't have a schema version in the entityset serialization format yet, so no need to worry about that. can you put up an issue though that we should add a schema version to entityset like we do feature serialization? 

2. Let's move it to `Relationship`. If we do that, should we move the deserialization method to relationship too?

3. I like the `to_dictionary` / `from_dictionary` naming convention that we use elsewhere

Reply:
That all sounds good.